apps/web/pages/signup.tsx

```diff
@@ -541,14 +542,14 @@ export default function Signup({
                       <Link
                         className="text-emphasis hover:underline"
                         key="terms"
-                        href={`${WEBSITE_URL}/terms`}
+                        href={`${WEBSITE_TERMS_URL}`}
                         target="_blank">
                         Terms
                       </Link>,
                       <Link
                         className="text-emphasis hover:underline"
                         key="privacy"
-                        href={`${WEBSITE_URL}/privacy`}
+                        href={`${WEBSITE_PRIVACY_POLICY_URL}`}
```

Comment:
I think it's okay, if this file is unchanged

Reply:
If this file is left unchanged, there is still the HTTP 404 error on self-hosted instances (see Signup page before - screenshot).

Reply:
Yeah, it's ok